src/main/java/uk/gov/hmcts/reform/pbis/notify/NotificationClientStub.java

```diff
@@ -0,0 +1,90 @@
+package uk.gov.hmcts.reform.pbis.notify;
+
+import java.util.Map;
+import uk.gov.service.notify.Notification;
+import uk.gov.service.notify.NotificationClientApi;
+import uk.gov.service.notify.NotificationClientException;
+import uk.gov.service.notify.NotificationList;
+import uk.gov.service.notify.SendEmailResponse;
+import uk.gov.service.notify.SendLetterResponse;
+import uk.gov.service.notify.SendSmsResponse;
+import uk.gov.service.notify.Template;
+import uk.gov.service.notify.TemplateList;
+import uk.gov.service.notify.TemplatePreview;
+
+public class NotificationClientStub implements NotificationClientApi {
+
+    private static final String UNSUPPORTED_OPERATION_MESSAGE =
+        "Notification client stub doesn't support this method";
```

Comment:
is this message really needed?

Reply:
Doesn't hurt. It might make it more obvious that the lack of support is deliberate.